CONTRIBUTION.md

```diff
@@ -58,7 +58,7 @@ git clone https://github.com/demisto/demisto-sdk.git
 
 ### 3. Pre-commit hooks setup
 We use are using [pre-commit](https://pre-commit.com/) to run hooks on our build. To use it run:
-1. Install hook to be performed as a hook before commiting changes - `pre-commit install`
+1. Install hook to be performed as a hook before commiting changes - `pip3 pre-commit install`
```

Comment:
It suppose to be that way

Reply:
It installed in https://github.com/demisto/demisto-sdk/blob/master/requirements-dev.txt